kernelci/kbuild.py

```diff
@@ -699,16 +699,17 @@ def submit(self, retcode, dry_run=False):
             },
             'child_nodes': [],
         }
-        # TODO: Fix this hack
+        # Update node from API, as we might have new fields
+        # such as k8s_context
+        nodeid = self._node['id']
+        self._node = api.node.get(nodeid)
         node = self._node.copy()
         results['node']['data'] = node['data']
-        results['node']['data']['kernel_revision'] = node['data']['kernel_revision']  # noqa
         results['node']['data']['arch'] = self._arch
         results['node']['data']['compiler'] = self._compiler
         results['node']['data']['defconfig'] = self._defconfig
         results['node']['data']['fragments'] = self._fragments
         results['node']['data']['config_full'] = self._config_full
-        results['node']['data']['dtb_present'] = dtb_present
```

Comment:
Why is this removed?

Reply:
This is not actually needed (`platform_config.dtb` is set for devices expecting a dtb, checking this field should be enough)

Reply:
I will add it to commit description, it was added shortly to assist @a-wai with ARM devices, but he found better way of detecting dtb.